content/fedramp.gov/README.md

```diff
@@ -1,3 +1,3 @@
 # Content Moved
 
-All OSCAL FedRAMP content files have been moved to the [OSCAL content GitHub repository](https://github.com/usnistgov/oscal-content/tree/master/fedramp.gov).
+All OSCAL FedRAMP content files have been moved to the [OSCAL content GitHub repository](https://github.com/usnistgov/oscal-content/tree/main/fedramp.gov).
```

Comment:
This link is an indirection. It would likely be better to point to https://github.com/GSA/fedramp-automation/tree/master/dist/content/baselines

Reply:
@bradh Thanks for pointing it out, fixed with a new commit :)